packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -177,29 +192,33 @@ class _BottomAppBarState extends State<BottomAppBar> {
     final Color color = widget.color ?? babTheme.color ?? defaults.color!;
     final Color surfaceTintColor = widget.surfaceTintColor ?? babTheme.surfaceTintColor ?? defaults.surfaceTintColor!;
     final Color effectiveColor = isMaterial3 ? color : ElevationOverlay.applyOverlay(context, color, elevation);
+    final Color? shadowColor = widget.shadowColor ?? babTheme.shadowColor ?? defaults.shadowColor;
```

Comment:
Maybe the M2 defaults should include the shadow color? If so, this would be `final Color ... deafults.shadowColor!`